tests/gorepo_test.go

```diff
@@ -1,15 +1,18 @@
-// +build !js
-
 package tests_test
 
 import (
 	"os"
 	"os/exec"
+	"runtime"
 	"testing"
 )
 
 // Go repository basic compiler tests, and regression tests for fixed compiler bugs.
 func TestGoRepositoryCompilerTests(t *testing.T) {
+	if runtime.GOARCH == "js" {
```

Comment:
What's the thinking behind dropping the `// +build !js` build tag here?

Reply:
The motivation is written in the commit message of d0d69c08b6af458cebb60d3295f643e4a471029d:

> Rely on runtime.GOARCH and t.Skip to skip tests that shouldn't be run
> with GopherJS. It's more clear and reliable than a file-wide build tag.

`t.Skip` is exists per-test (rather than for the entire file) and can offer an individual message for the skip reason. This was better in this context. Build tags were used previously because I didn't get the idea that `t.Skip` could be used instead until now.